staging/src/k8s.io/apiserver/pkg/server/options/admission_test.go

```diff
@@ -36,7 +36,7 @@ func TestEnabledPluginNames(t *testing.T) {
 	}{
 		// scenario 0: check if a call to enabledPluginNames sets expected values.
 		{
-			expectedPluginNames: []string{"NamespaceLifecycle", "MutatingAdmissionWebhook", "ValidatingAdmissionWebhook"},
+			expectedPluginNames: []string{"NamespaceLifecycle", "MutatingAdmissionWebhook", "ValidatingAdmissionPolicy", "ValidatingAdmissionWebhook"},
```

Comment:
I think that there is another "CEL" string constant somewhere in the recent changes (Alex's PR?). Update it as well?

Reply:
nevermind, I see that it got fixed.